app/static/src/app/plot.ts

```diff
@@ -19,7 +19,8 @@ export const margin = {
 };
 
 export const config = {
-    responsive: true
+    responsive: true,
+    displayModeBar: true
```

Comment:
has this PR changed the plotly config to remove the floating bar (despite this saying true)? People use that to download plots so it's important to retain. I see it's there on the version on wodin-dev still at least

Reply:
ahh good point, it was setting it as false because i thought it would have controls for zoom, allowing users to bypass the zoom lock when axes are locked, however plotly is smart and if i lock the axes then the zoom options dont show up! so it is added back in